extensions/open-remote-ssh/src/authResolver.ts

```diff
@@ -244,13 +244,15 @@ export class RemoteSSHResolver implements vscode.RemoteAuthorityResolver, vscode
 				if (context.resolveAttempt === 1) {
 					this.logger.show();
 
-					const closeRemote = 'Close Remote';
```

Comment:
Since this is a hard fork, we don't need code fences, right? Happy to add them if you think it's a good idea.

Reply:
Right, no need for code fences in this file.